pages/common/rich.md

```diff
@@ -0,0 +1,32 @@
+# rich
+
+> Rich CLI - fancy terminal display.
+> More information: <https://github.com/Textualize/rich-cli>.
+
+- Show file contents with syntax highlighting:
```

Comment:
```suggestion
- Show a specific file with syntax highlighting:
```

Reply:
@navarroaxel, what about to stick to `Display` word instead of `Print` in all pages?